configs/analysis_configs/cytotable_convert.yaml

```diff
@@ -0,0 +1,11 @@
+cytotable_convert:
+  params:
+    dest_datatype: parquet
+    source_datatype: null
```

Comment:
Ah yes, It definitely does! 

The way how these configs files are designed is that we take the default parameters of a function and place them into the config file. 

Looking at [CytoTable’s](https://github.com/cytomining/CytoTable/blob/main/cytotable/convert.py) `convert` workflow, it seems that `source_datatype` is set to `None`. However, you are correct. It is more practical for the configs to have their own defaults. Like in this case, making `source_datatype` as `sqlite` instead of `null` since the expected inputs are `sqlite` files.

Thanks for pointing this out!